src/status_im/components/tabs/styles.cljs

```diff
@@ -1,29 +1,24 @@
 (ns status-im.components.tabs.styles
-  (:require [status-im.components.styles :as st]
+  (:require-macros [status-im.utils.styles :refer [defnstyle defstyle]])
+  (:require [status-im.components.styles :as common-styles]
```

Comment:
What about using `styles` ? I started to migrate to it.

Reply:
do you mean `styles` alias instead `common-styles` ?

Reply:
Yes